src/documentation.js

```diff
@@ -229,7 +229,7 @@ Documentation.Class = class {
       const member2 = this.membersArray[i + 1];
       if (member1.kind !== 'event' || member2.kind !== 'event')
         continue;
-      if (member1.name > member2.name)
+      if (member1.name.localeCompare(member2.name, 'en', { sensitivity: 'base' }) > 0)
```

Comment:
member1.name is always ASCII, let's keep it simple and consistent with upstream documentation.js

Reply:
I did just change the upstream one to use this same approach, though. I'm all for simplicity, but... let's keep it this way for now. :-)